### PR TITLE
[Commander]: Copy over Chromium required Chromium files

### PR DESCRIPTION
**Pull request by fallaciousreasoning**
Chromium change:
https://chromium.googlesource.com/chromium/src/+/02d83d4c22496391174a0777976a49f984081f95

Resolves

## Submitter Checklist:

- [ ] I confirm that no [security/privacy review is needed](https://github.com/brave/brave-browser/wiki/Security-reviews) and no other type of reviews are needed, or that I have [requested](https://github.com/brave/reviews/issues/new/choose) them
- [ ] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [ ] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [ ] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally:
  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

File: chromium_src/chrome/browser/ui/cocoa/main_menu_builder.mm [hunk 1]
```diff
@@ -37,6 +37,8 @@
 IDS_TAB_CXMENU_CLOSE_DUPLICATE_TABS).command_id(IDC_CLOSE_DUPLICATE_TABS), \
               Item(kCloseOtherTabsResourceId
 
+// TODO(fallaciousreasoning): Add menu item for COMMANDER
```
Comment (fallaciousreasoning) [inline chromium_src/chrome/browser/ui/cocoa/main_menu_builder.mm:40]: @cdesouza-chromium I don't have a Mac, so I'm a little scared to do this.

Comment (cdesouza-chromium) [reply chromium_src/chrome/browser/ui/cocoa/main_menu_builder.mm:40]: I'm not sure if I follow what you're trying to say here. I do have a mac if you want me to test something for you though.

File: browser/ui/commander/bookmark_command_source.cc [hunk 2]
```diff
@@ -0,0 +1,89 @@
+// Copyright 2020 The Chromium Authors
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
```
Comment (emerick) [inline browser/ui/commander/bookmark_command_source.cc:3]: We should update all of these header comments so the linter doesn't trip over them.

File: chromium_src/chrome/browser/ui/cocoa/main_menu_builder.mm (hunk 1 above)
Comment (emerick) [reply chromium_src/chrome/browser/ui/cocoa/main_menu_builder.mm:40]: This is only an issue for Mac? What about Windows/Linux?

File: browser/ui/commander/bookmark_command_source.cc (hunk 2 above)
Comment (fallaciousreasoning) [reply browser/ui/commander/bookmark_command_source.cc:3]: Ah yeah, so I wasn't sure how to handle those - I don't know what we normally do, but I think the copyright probably still belongs to The Chromium Authors?

File: chromium_src/chrome/browser/ui/cocoa/main_menu_builder.mm (hunk 1 above)
Comment (fallaciousreasoning) [reply chromium_src/chrome/browser/ui/cocoa/main_menu_builder.mm:40]: For Windows/Linux, the menu item is added here:
https://github.com/brave/brave-core/pull/22107/files#diff-2283271e14350311184ccfe17b67de5e026532933dd04faea092caa3611f4871R390

Was a little hesitant to add the menu item back on MacOS, and wasn't 100% sure where it should appear.

File: chromium_src/chrome/browser/ui/views/accelerator_table.cc [hunk 3]
```diff
@@ -5,7 +5,11 @@
 
 #include "chrome/browser/ui/views/accelerator_table.h"
 
+#include "brave/app/brave_command_ids.h"
+#include "brave/components/commander/common/buildflags/buildflags.h"
 #include "build/build_config.h"
+#include "ui/events/event_constants.h"
+#include "ui/events/keycodes/keyboard_codes_posix.h"
```
Comment (fallaciousreasoning) [inline chromium_src/chrome/browser/ui/views/accelerator_table.cc:12]:
```suggestion
```

**Comment by fallaciousreasoning** [comment]
Closing, @emerick cherry-picked the commits.

File: chromium_src/chrome/browser/ui/cocoa/main_menu_builder.mm (hunk 1 above)
Comment (sangwoo108) [reply chromium_src/chrome/browser/ui/cocoa/main_menu_builder.mm:40]: This seems to be where it was

image

Restored here https://github.com/brave/brave-core/pull/22543

**Pull request closed without merging**